mobile/build.gradle

```diff
@@ -170,10 +169,16 @@ dependencies {
 
     testCompile project(':testlib')
     testCompile 'org.mockito:mockito-core:1.10.8'
+    compile('com.crashlytics.sdk.android:crashlytics:2.2.0@aar') {
+        transitive = true;
```

Comment:
It appears that the `transitive  = true` is necessary. Without it the app crashes immediately.